lib/core/public/run.js

```diff
@@ -66,7 +66,6 @@ export default function run(...args) {
       axe.utils.performanceTimer.end();
     }
     axe._running = false;
-    resetGlobals();
```

Comment:
This is already called in `teardown`, which is invoked earlier. Calling `resetGlobals` here does nothing, and wouldn't actually have worked because axe's cache has been cleaned up by this point. Same is true for the other `resetGlobals()` call in this file.